github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Kindly change the containing folder of your `linters.yml` file from `workflow` to `workflows`  because `workflow` is the wrong spelling and this is causing your linters to not show up on Github
- [ ] Please kindly ensure all your linters are passing on Github i.e they should all have a green :heavy_check_mark:

Reply:
Kindly implement the above-suggested ☝🏽  changes. This will help GitHub workflows to assess your code for conformity and either ✔️ or ❌. This is a requirement for this project to be approved. Currently, your directory structure for linters is `github/workflow/linters.yml` while the correct directory is `.github/workflows/linters.yml`. Check and make sure your directory is exact.